irteus/test/test-irt-motion.l

```diff
@@ -871,6 +871,48 @@
                                     (send robot :calc-static-balance-point :force-list (list #f(10 20 0) #f(25 20 0))))))
     ))
 
+;; test for static balance point
+(defun fullbody-ik-with-forces-moments (robot &key (debug-view :no-message))
+  (send robot :newcoords (make-coords))
+  (send robot :reset-pose)
+  (send robot :fix-leg-to-coords (make-coords))
+  (send robot :legs :move-end-pos (float-vector 0 0 50))
+  (send robot :fix-leg-to-coords (make-coords))
+  (objects (list robot))
+  (let* ((centroid-thre 10)
+         (force-list '(#f(10 20 0) #f(25 20 0)))
+         (moment-list '(#f(20 -10 0) #f(10 -20 0)))
+         (result
+          (send robot :fullbody-inverse-kinematics
+                (list (send robot :rleg :end-coords :copy-worldcoords)
+                      (send robot :lleg :end-coords :copy-worldcoords)
+                      (make-coords :pos #f(150 -300 600))
+                      (make-coords :pos #f(150 300 600)))
+                :move-target (mapcar #'(lambda (x)
+                                         (send robot x :end-coords))
+                                     (list :rleg :lleg :rarm :larm))
+                :link-list (mapcar #'(lambda (x)
+                                       (send robot :link-list (send robot x :end-coords :parent)))
+                                   (list :rleg :lleg :rarm :larm))
+                :centroid-thre centroid-thre
+                :centroid-offset-func #'(lambda () (send robot :calc-static-balance-point :force-list force-list :moment-list moment-list))
+                :debug-view debug-view)))
+    (and result
+         (> centroid-thre (distance (apply #'midpoint 0.5 (send robot :legs :end-coords :worldpos))
+                                    (send robot :calc-static-balance-point :force-list force-list :moment-list moment-list)))
+         (> 1.0
```

Comment:
Please add a comment about the meaning of this condition checking.

Reply:
I follow this code, and now understand this condition check and think this is OK. (But still add comment please.)